frontend/src/pages/RecipientRecord/pages/GoalsObjectives.js

```diff
@@ -27,10 +27,8 @@ export default function GoalsObjectives({
   const onRemoveFilter = (id) => {
     const newFilters = [...filters];
     const index = newFilters.findIndex((item) => item.id === id);
-    if (index !== -1) {
```

Comment:
these were removed to raise branch coverage; it may look a little weird, but when I set about writing a test for it, there was no way to do it through the UI, so not sure it makes sense to check here